cli/tools/test/mod.rs

```diff
@@ -399,6 +399,33 @@ impl TestFailure {
     }
   }
 
+  pub fn error_location(&self) -> Option<TestLocation> {
+    match self {
+      // TODO: What happens if not enough call stack frames were recorded?
```

Comment:
Turns out that this is a very real issue. If we have about 10 extra call stack frames above the test, then we no longer have enough info to figure out where in the test the error happened.

e.g
```ts
Deno.test("database operations", (t) => {
  throwDeep(10)
});

function throwDeep (d = 0) {
  if(d <= 0) {
  throw new Error("bar");
  } else {
    throwDeep(d - 1)
  }
}
```

Reply:
This makes implementing a correct "line in test file where error was thrown" significantly harder. 

I initially thought that I could skip all Deno internal frames from the bottom. Logically, the first thing after that must be a user frame.

Except that our stack trace might be cut off. In which case it sometimes doesn't have any Deno internal frames.

Or worse yet, the stack might randomly cut off after a completely unrelated Deno internal frame. 
e.g. `Deno.inspect` is an internal function. And we can throw errors from within a `[Symbol.for("Deno.customInspect")]`